src/servers/ZoneServer/zoneserver.ts

```diff
@@ -795,7 +806,7 @@ export class ZoneServer extends EventEmitter {
 
       this.sendData(
         client,
-        "PlayerUpdate.RemovePlayerGracefully",
```

Comment:
Why changing that? With this packet the entity is still in the game i don't think we should use it

Reply:
RemovePlayer packet for less resource usage, if it doesnt work that way ima change it back